### PR TITLE
Sociallogin

### PR DESCRIPTION
**Pull request by FsDevNinja**
Working on Facebook login integration with omniauth.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
-
+  
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
-
+  
   # You can have the root of your site routed with "root"
   # root 'welcome#index'
-
+
```
Comment (lasley) [inline config/routes.rb]: FYI the more changes you make to the diff, the more complicated your life is during a conflict. With that in mind, I don't recommend stylistic changes such as this without an explicit reason.

File: .gitignore [hunk 2]
```diff
@@ -14,3 +14,6 @@
 # Ignore all logfiles and tempfiles.
 /log/*.log
 /tmp
+
+# Ignore application configuration
+/config/application.yml
```
Comment (lasley) [inline .gitignore]: You're adding this to the gitignore, but it's already checked into source control. If intent is to actually ignore it, will also need to `git rm`

File: config/application.yml [hunk 3]
```diff
@@ -12,4 +12,4 @@
 
 FACEBOOK_KEY: "1766735706910742"
 FACEBOOK_SECRET: "6f04fbd8f7c80dd2dbf5d62098afb8d6"
```
Comment (lasley) [inline config/application.yml]: Change your Facebook secret immediately. It is now compromised

**Comment by PamBWillenz** [comment]
Looks good, Matt - great job. I didn't see a twitter spec in your spec test - did you leave it out for a reason?

**Comment by FsDevNinja** [comment]
No, I forgot twitter. it is in now.

**Comment by lasley** [comment]
This was merged with 0% test coverage 😦

**Comment by lasley** [comment]
Oh wait sorry I lied, I think I need more ☕️ . GJ!

**Comment by PamBWillenz** [comment]
Oh no - I saw Matt's omniauth controller spec - wait just saw your second note - is everything cool?

**Comment by lasley** [comment]
Yup it's fine from a high level - We won't truly know until (or if) we add coverage results.

**Pull request merged**